Update submit_jobs.py

yizhangcern opened the pull request:
Updated CaloParems to 2018_v1_3 and made the CaloParams tag optional, e.g. '-c 2018_v1_1_ECALZS'

yizhangcern committed:
Update submit_jobs.py

Updated CaloParems to 2018_v1_3 and made the CaloParams tag optional, e.g. '-c 2018_v1_1_ECALZS'

christopheralanwest commented on scripts/submit_jobs.py:
```diff
@@ -12,7 +12,8 @@
 CONDITIONS = '101X_dataRun2_HLT_v7'
 # L1 calibrations; needs to be updated when L1 calibrations change
-CALOSTAGE2PARAMS = '2018_v1_1_ECALZS'
+#CALOSTAGE2PARAMS = '2018_v1_1_ECALZS'
```
Please remove this commented line. We will likely never use this tag again and so there's no need to keep it here. If someone really wants to know the history of the script, they can look at the revision log.

christopheralanwest commented on scripts/submit_jobs.py:
```diff
@@ -66,6 +67,7 @@ def generate_ntuple_config(configtype, newtag):
 PARSER.add_argument('-d', '--dataset', required=True)
 PARSER.add_argument('-o', '--outputsite', required=True)
 PARSER.add_argument('-n', '--no_exec')
+PARSER.add_argument('-c', '--caloparams')
```
The `customise` command at [L50](https://github.com/cms-hcal-trigger/Validation/pull/4/files#diff-d2f446b0e82f32b776ed125168954504R50) should only be used if this argument is defined. It should no longer be necessary to explicitly specify the CaloParams, as they are now taken from the global tag, so [L50](https://github.com/cms-hcal-trigger/Validation/pull/4/files#diff-d2f446b0e82f32b776ed125168954504R50) would override the CaloParams from the GT. You should check that you get exactly identical results if the CaloParams are specified explicitly or if they are taken from the global tag.

christopheralanwest replied:
Could you please follow up on this comment?

christopheralanwest replied:
Sorry, somehow I missed that you had already implemented the requested changes.

yizhangcern committed:
Update submit_jobs.py

Suppose that we would get exactly identical results if the CaloParams are specified explicitly or if they are taken from the global tag. Then the customise command at L50 would only be used when argument  --caloparams is defined. 
CaloParams tag is optional, e.g. '-c 2018_v1_1_ECALZS'
